<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BirimVeriGirisi</class>
 <widget class="QWidget" name="BirimVeriGirisi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>230</y>
     <width>279</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="0">
     <widget class="QPushButton" name="btnSil">
      <property name="text">
       <string>IPTAL</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnKaydet">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>541</width>
     <height>338</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="lblBirimKodu">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Birim Kodu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leBirimKodu"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblAdresi">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Adresi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="leTelefon"/>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="teAdresi"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblTelefon">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telefon</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leBirimAdi"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblBirimAdi">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Birim Adi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>276</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizeIncrement">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../Kaynaklar.qrc">:/Resimler/Birim48.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblBaslik">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Yeni Yesilay Girisi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSil</sender>
   <signal>clicked()</signal>
   <receiver>BirimVeriGirisi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
